app/page.tsx

```diff
@@ -2,112 +2,766 @@ import Image from "next/image";
+      text-white bg-black sm:w-auto dark:bg-black dark:hover:bg-gray-900">Get in touch</button>
+                  </form>
+                </div>
+              </div>
+              </div>
+            </div>
+
+            <div className="container mx-auto flex flex-wrap py-5 flex-col md:flex-row items-center">
+              <div className="">
+                <img
+                  src="images/first.png"
+                  className="h-auto max-w-full"
+                  alt="..." />
+              </div>
+            </div>
+
+            {/* about page */}
+            <div className="container mx-auto flex flex-wrap py-16 flex-col md:flex-row items-center">
+              <section className="text-gray-900 body-font">
+                <div className="container px-5 py-23 mx-auto ">
+                  <div className=" w-full px-28 mx-auto">
+                    <h1 className=" font-bold pb-8 text-2xl">About Project</h1>
+                    <p className="leading-relaxed text-xl">
+                      Welcome to Trusti, your all-in-one financial assistant! We offer real-time insights,
+                      seamless budgeting, and expert investment advice to help you manage your finances effortlessly.
+                      With features like Smart Insights, Budget Buddy, and Growth Forecaster, you’ll receive personalized
+                      recommendations tailored to your financial goals. Stay secure with our Security Shield and expand your
+                      knowledge with our Learning Hub. Monitor your transactions easily and keep track of your savings progress
+                      with our intuitive tools. Join Trusti today and start your journey to financial clarity and growth!
+                    </p>
+                  </div>
+                </div>
+              </section>
+
+            </div>
+
+            {/* second picture */}
+            <div className="container mx-auto flex flex-wrap flex-col md:flex-row items-center">
+              <div className="">
+                <img
+                  src="images/second.png"
+                  className="h-auto max-w-full"
+                  alt="..." />
+              </div>
+            </div>
+
+            {/* third picture */}
+            <div className="container mx-auto mt-16 flex flex-wrap flex-col md:flex-row items-center">
+              <div className="">
+                <img
+                  src="images/third.png"
+                  className="h-auto max-w-full"
+                  alt="..." />
+              </div>
+            </div>
+
+
+            {/* second picture */}
+            <div className="container mx-auto flex mt-16 flex-wrap flex-col md:flex-row items-center">
+              <div className="">
+                <img
+                  src="images/second.png"
+                  className="h-auto max-w-full"
+                  alt="..." />
+              </div>
+            </div>
+
```

Comment:
```suggestion
            {/* second picture */}
            <div className="container mx-auto flex flex-wrap flex-col md:flex-row items-center">
              <div className="">
                <img
                  src="images/second.png"
                  className="h-auto max-w-full"
                  alt="..." />
              </div>
            </div>

            {/* third picture */}
            <div className="container mx-auto mt-16 flex flex-wrap flex-col md:flex-row items-center">
              <div className="">
                <img
                  src="images/third.png"
                  className="h-auto max-w-full"
                  alt="..." />
              </div>
            </div>


            {/* second picture */}
            <div className="container mx-auto flex mt-16 flex-wrap flex-col md:flex-row items-center">
              <div className="">
                <img
                  src="images/second.png"
                  className="h-auto max-w-full"
                  alt="..." />
              </div>
            </div>

```

Convert all of this into a component for reusability